ruma-client-api/src/r0/message/send_message_event.rs

```diff
@@ -1,7 +1,11 @@
 //! [PUT /_matrix/client/r0/rooms/{roomId}/send/{eventType}/{txnId}](https://matrix.org/docs/spec/client_server/r0.6.1#put-matrix-client-r0-rooms-roomid-send-eventtype-txnid)
 
-use ruma_api::{ruma_api, Metadata};
-use ruma_events::{AnyMessageEventContent, EventContent as _};
+use ruma_api::ruma_api;
+#[cfg(any(feature = "client", feature = "server"))]
+use ruma_api::Metadata;
```

Comment:
I really don't like these conditional imports. Can you instead update the `METADATA` const to use a qualified path in the type?